ota-plus-web/app/com/advancedtelematic/controllers/Application.scala

```diff
@@ -109,25 +109,13 @@ class Application @Inject() (ws: TraceWSClient,
     case (_, "assignments" :: _) => directorApiUri
   }
 
-  private val coreProxiedPrefixes: Dispatcher = {
-    case (_, "packages" :: _) => coreApiUri
-    case (_, "devices_info" :: _) => coreApiUri
-    case (_, "device_updates" :: _) => coreApiUri
-    case (_, "vehicle_updates" :: _) => coreApiUri
-    case (_, "update_requests" :: _) => coreApiUri
-    case (_, "history" :: _) => coreApiUri
-    case (_, "blacklist" :: _) => coreApiUri
```

Comment:
I found only
```
  case (_, "blacklist" :: _) => coreApiUr
  case (_, "impact" :: _) => coreApiUri
```
where used. The first one is going to device-registry now, and the second one is replaced with `GET /api/v1/blacklist`, again in device-registry.